### PR TITLE
Nav updates

### PR DESCRIPTION
**Pull request by newsummit**
- Styling navbar and sub-navbar updates for cleaner rendering in prod.
- Styling the power-select fields for self-serve manage alerts feature.
- Prepping search results page

screen shot 2017-06-18 at 12 25 42 pm

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/application/controller.js [hunk 1]
```diff
@@ -33,6 +34,7 @@ export default Ember.Controller.extend({
       className: 'alert',
       link: 'self-service',
       title: 'Self-Service'
+      //hidden: true
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/pods/application/controller.js]: Toggle this on to hide the item

File: thirdeye/thirdeye-frontend/app/pods/application/template.hbs [hunk 2]
```diff
@@ -1,6 +1,2 @@
 {{te-navbar navItems=globalNavItems isGlobalNav=true navClass="te-nav--global"}}
-
-<div class="paper-container container">
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/application/template.hbs]: Can you make sure that the rca/metrics page still look ok after this change?

File: thirdeye/thirdeye-frontend/app/pods/self-service/manage/template.hbs [hunk 3]
```diff
@@ -1,48 +1,64 @@
-<h1 class="te-title">Manage Alerts</h1>
-{{outlet}}
+<div class="te-title">Manage Alerts</div>
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/self-service/manage/template.hbs]: I think `h1` might be more semantic. I know we might have to override a lot of bootstrap classes, perhaps we could do it in one mixin (@include reset-bootstrap-title) for all h* ?

File: thirdeye/thirdeye-frontend/app/styles/app.scss [hunk 4]
```diff
@@ -38,7 +29,7 @@ body {
 @import 'components/te-navbar';
 
 // Pod Pages
-@import 'main';
 @import 'pods/rca';
-
+@import 'ember-power-select/themes/bootstrap';
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/styles/app.scss]: 👍

File: thirdeye/thirdeye-frontend/app/pods/application/template.hbs (hunk 2 above)
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/pods/application/template.hbs]: Done. Looks good.

File: thirdeye/thirdeye-frontend/app/pods/self-service/manage/template.hbs (hunk 3 above)
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/pods/self-service/manage/template.hbs]: Sure, can do.

**Pull request merged**